app/controllers/applications/supplementals_controller.rb

```diff
@@ -3,9 +3,11 @@ module Applications
   class SupplementalsController < ApplicationController
     before_action :authenticate_user!
     def index
-      @application = application_service.application(params[:application_id])
+      includes = %w[preferences proof_files household_members flagged_applications lease]
+      @application = application_service.application(params[:application_id], includes: includes)
       @status_history = field_update_comment_service.status_history_by_application(params[:application_id])
       @file_base_url = file_base_url
+      @available_units = units_service.available_units_for_application(@application.Listing.Id, @application.Id)
       @units = listing_service.units(@application.Listing.Id)
```

Comment:
does it make sense to move the units over to the Units_service too? Is there a better name for units to clarify the difference between available units and listing units?

Reply:
Ye, we could move it. However, this might be out of the scope this story.